scripts/projector.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# failfast
+set -e
+
+# Define these 2 optional variables if you want to customize the default installation,
+# then copy-paste the remainder of the script:
+# - the name of the projector config (defaults to 'Idea')
+# projector_config_name=Idea
+# - the name of the IDE to install (defaults to "IntelliJ IDEA Community Edition 2020.3.4")
+# Note: to get list of available IDEs, launch `projector find`
+# projector_ide_name="IntelliJ IDEA Community Edition 2020.3.4"
+
+projector_config_name=${projector_config_name:-Idea}
+projector_ide_name=${projector_ide_name:-IntelliJ IDEA Community Edition 2020.3.4}  # Latest projector-tested IDE
+
+# Install projector-installer
+# https://github.com/JetBrains/projector-installer
+apt-get update
+apt-get install -y --no-install-recommends python3 python3-pip python3-cryptography less libxext6 libxrender1 libxtst6 libfreetype6 libxi6
+pip3 install projector-installer
+
+# Setup projector
+sudo -u ubuntu projector --accept-license ide autoinstall --config-name "${projector_config_name}" --ide-name "${projector_ide_name}" --hostname {{ .STRIGO_RESOURCE_DNS }}
+# Cannot use password as it needs to be in the URL as query parameter 'token'
+# and it cannot be different for each session in Strigo web interfaces
+# cat << EOF >> "~ubuntu/.projector/configs/${projector_config_name}/config.ini"
+# [PASSWORDS]
+# password = {{ .STRIGO_WORKSPACE_ID }}
+# ro_password = {{ .STRIGO_WORKSPACE_ID }}
+# EOF
+# sudo -u ubuntu projector --accept-license config rebuild "${projector_config_name}"
+
+# Create, enable and start projector config service
+cat << EOF > /lib/systemd/system/projector@.service
+[Unit]
+Description=Jetbrains Projector - ${projector_config_name}
+After=network.target
+
+[Service]
+Type=exec
+ExecStart=/home/ubuntu/.projector/configs/${projector_config_name}/run.sh
+Restart=always
+User=%i
+
+[Install]
+WantedBy=default.target
+EOF
+systemctl enable --now projector@ubuntu
+
+# Display projector config password in terminal
+# cat <<\EOF > /etc/profile.d/projector-terminal.sh
+# if [ $USER = ubuntu ]; then
+#   echo -ne '\nProjector '
+#   grep '^password = ' "~/.projector/configs/${projector_config_name}/config.ini"
+#   echo
+# fi
+# EOF
```

Comment:
why are these lines commented out?

Reply:
Removed as it has no real sense because the password has to be the same for all sessions so will be provided in Strigo button and/or trainer.